github/repos_contents_test.go

```diff
@@ -54,6 +54,12 @@ func TestRepositoryContent_GetContent(t *testing.T) {
 			want:     "",
 			wantErr:  true,
 		},
+		{
+			encoding: String("none"),
+			content:  String(""),
```

Comment:
```suggestion
```

content will be nil when encoding is none, so leaving it nil here is a slightly more realistic test case.

Reply:
good catch!